apps/web-giddh/src/app/shared/header/header.component.html

```diff
@@ -461,40 +461,38 @@ <h4 class="inline text-capitalize" id="giddh-page-heading" *ngIf="!navigationEnd
                     </ul>
-                </div>
-
-                <div class="lower-layer-header">
-                    <ul class="nav universal-date  pull-left" *ngIf="(selectedCompany | async)">
-                        <li class="border-none pl-0">
-
-                            <button class="btn btn-link" *ngIf="!isTodaysDateSelected" (click)="dateSelectedCallback()">Today</button>
-                            <!-- <button class="btn btn-link" *ngIf="isTodaysDateSelected" (click)="showGiddhDatepicker($event)">Today</button> -->
-                        </li>
-                        <li class="daterang-picker not-today cursor-pointer">
-                            <span class="icon-calender calender-addone" (click)="showGiddhDatepicker($event)"></span>
-                            <div class="input-date" [ngClass]="{'today-date': isTodaysDateSelected}">
-                                <input type="text" name="selectedDateRange" (click)="showGiddhDatepicker($event)"
-                                    [value]="(selectedDateRangeUi) ? selectedDateRangeUi : ''"
-                                    class="giddh-datepicker form-control date-range-picker" />
-                            </div>
-                        </li>
-                    </ul>
                     <div class="d-flex align-items-center" *ngIf="(selectedCompany | async)">
                         <span [ngClass]="{'notification': isMobileScreen}">
                             <!-- <img src="{{imgPath}}notification.svg" class="mr-15"> -->
                             <i class="icon-notification mr-15"></i>
                         </span>
-                        <button id="add-manage" *ngIf="!isMobileSite" type="button" class="btn btn-gray btn-sm d-inline-flex mr-15 " [checkPermission]="['MENU', 'MANAGE']" (click)="showManageGroupsModal()">
+                        <button id="add-manage" *ngIf="!isMobileSite" type="button" class="btn btn-gray btn-sm d-inline-flex mr-15 "
+                            [checkPermission]="['MENU', 'MANAGE']" (click)="showManageGroupsModal()">
                             <i class="icon-plus"></i>
                         </button>
                         <div class="btn-group cp d-inline-flex help-setting-icons" dropdown>
                             <span (click)="toggleHelpSupportPane(true)">
-                                <img src="{{imgPath}}help-support-icon.svg">
+                                <i class="icon-help-support-icon"></i>
                             </span>
                         </div>
 
                     </div>
                 </div>
+
+                <div class="lower-layer-header">
+                    <ul class="nav universal-date  pull-left" *ngIf="(selectedCompany | async)">
+                        <li class="border-none pl-0" [ngClass]="{'hide-today':isTodaysDateSelected}">
+                            <button class="btn btn-link" (click)="dateSelectedCallback()"> Today</button>
```

Comment:
@shalinee-walkover you have removed *ngIf="!isTodaysDateSelected" 
is this correct?
 please check all possible scenario